Common/Properties/SharedAssemblyInfo.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System.Reflection;
+using System.Runtime.CompilerServices;
 
 // common assembly attributes
 [assembly: AssemblyDescription("Lean Algorithmic Trading Engine - QuantConnect.com")]
 [assembly: AssemblyCopyright("Copyright ©  2015")]
 [assembly: AssemblyCompany("QuantConnect")]
 [assembly: AssemblyTrademark("QuantConnect")]
 [assembly: AssemblyVersion("2.2.0.2")]
+
+[assembly: InternalsVisibleTo("QuantConnect.Tests")]
```

Comment:
This has been added for BrokerageTransactionHandler message pump to be visible outside but if you want to make `HandleOrderRequest()` public it can be removed.